src/3d/qgs3dmapscene.h

```diff
@@ -0,0 +1,76 @@
+#ifndef QGS3DMAPSCENE_H
```

Comment:
Is it voluntary that there isn't the usual header (email, copyright, and so on) in new .h/.cpp files?